scripts/imgtool.py

```diff
@@ -1,13 +1,27 @@
 #! /usr/bin/env python3
 
 import argparse
+import getpass
 from imgtool import keys
 from imgtool import image
 from imgtool import version
 import sys
 
 def gen_rsa2048(args):
-    keys.RSA2048.generate().export_private(args.key)
+    if args.password:
+        while True:
+            passwd = getpass.getpass("Enter key passphrase: ")
+            passwd2 = getpass.getpass("Reenter passphrase: ")
+            if passwd == passwd2:
```

Comment:
This is OK but maybe in the future would be interesting checking for a minimum size, and maybe using `passwordmeter` or some other entropy measuring library...